python/lsst/ts/audiotrigger/constants.py

```diff
@@ -19,11 +19,14 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-__all__ = ["MAX_PI_GPIOS", "SLEEP", "THRESHOLD"]
+__all__ = ["MAX_PI_GPIOS", "SLEEP", "THRESHOLD", "MIN", "MAX", "SENSOR_INDEX"]
 
 MAX_PI_GPIOS = 32
 """Maximum number of pi GPIO pins."""
 SLEEP = 1
 """Default sleep timer."""
 THRESHOLD = 10
 """Threshold over which misalignment of laser beam occurs."""
+MIN = 19
+MAX = 25
+SENSOR_INDEX = 0
```

Comment:
Please add doc strings for these variables. Also, `MIN` and `MAX` are very generic words. Please rename them to something meaningful.